src/main/java/duke/Instruction.java

```diff
@@ -0,0 +1,31 @@
+package duke;
+
+/**
+ * Stores instructions given by user.
+ */
```

Comment:
Perhaps you could add some elaborate documentation for each instruction for future reference?